internal/service/rds/cluster_data_source.go

```diff
@@ -136,11 +137,12 @@ func DataSourceCluster() *schema.Resource {
 }
 
 func dataSourceClusterRead(d *schema.ResourceData, meta interface{}) error {
+	ctx := context.TODO()
```

Comment:
Yould change `Read` to `ReadWithoutTimeout` for this data source and pass through the true `Context`.

Reply:
Good point